Yousei.Core/SimpleConnection.cs

```diff
@@ -13,14 +13,14 @@ public class SimpleConnection : IConnection
 
         private readonly Dictionary<string, IFlowTrigger> triggers = new();
 
-        public IFlowAction CreateAction(string name)
+        public IFlowAction? CreateAction(string name)
```

Comment:
When *could* this become a null? I don't get this. :/

Reply:
It returns `default` when no action is found in `actions`. The same goes for the triggers.